React/Views/RCTPickerManager.m

```diff
@@ -19,7 +19,9 @@ @implementation RCTPickerManager
 
 - (UIView *)view
 {
-  return [RCTPicker new];
+  RCTPicker *picker = [RCTPicker new];
+  [picker selectRow:0 inComponent:0 animated:YES];
```

Comment:
whats different than before? we are not configuring anything here, im just setting some defaults so the lines appear.